docs/basic-features/environment-variables.md

```diff
@@ -76,6 +76,9 @@ export async function getStaticProps() {
 > CORRECT=pre\$A
 > ```
 
+> **Note**: If you are using a `/src` folder, please note that Next.JS will load the .env files **only** from the parent
+> folder and **not** from the `/src` folder.
```

Comment:
```suggestion
```

Reply:
Let's keep the word "folder".

Reply:
I moved it up to the line above, unfortunately couldn't do a multi-line change with the review tool.